docs/sources/alerting/fundamentals/notifications/group-alert-notifications.md

```diff
@@ -69,22 +69,117 @@ If you want to receive every alert as a separate notification, you can do so by
+| 01:30              | `alert_name=f3` `team=frontend` | `frontend`                | 3                   |                                                                                                |
+| 02:30              | `alert_name=f4` `team=frontend` | `frontend`                | 4                   |                                                                                                |
+| 05:30<sup>\*</sup> |                                 | `frontend`                | 4                   | Group interval elapsed and resets timer. <br/> Send one notification reporting 4 alerts.       |
+| 05:50<sup>\*</sup> |                                 | `backend`                 | 3                   | Group interval elapsed and resets timer. <br/> No group changes, and do not send notification. |
+| 08:00              | `alert_name=f4` `team=backend`  | `backend`                 | 4                   |                                                                                                |
+| 10:30<sup>\*</sup> |                                 | `frontend`                | 4                   | Group interval elapsed and resets timer. <br/> No group changes, and do not send notification. |
+| 10:50<sup>\*</sup> |                                 | `backend`                 | 4                   | Group interval elapsed and resets timer. <br/> Send one notification reporting 4 alerts.       |
+
+**How it works**
+
+Once the first notification has been sent for a new group of alerts, the Group interval timer starts.
+
+When the Group interval timer elapsed, the system resets the Group interval timer and sends a notification only if there were group changes. This process repeats until there are no more alerts.
+
+It's important to note that an alert instance exits the group after being resolved and notified of its state change. When no alerts remain, the group is deleted, and then the Group wait timer handles the first notification for the next incoming alert once again.
+
+### Repeat interval
+
+**Default**: 4 hours
+
+Repeat interval acts as a reminder that alerts in the group are still firing.
+
+The Repeat interval timer decides how often notifications are sent (or repeated) if the group has not changed since the last notification.
+
+**How it works**
+
+Repeat interval is evaluated every time the Group interval resets. If the alert group has not changed and the time since the last notification was longer than the repeat interval, then a notification is sent to remind the alerts are still firing.
+
+Repeat interval must not only be greater than or equal to Group interval, but also must be a multiple of Group interval. If Repeat interval is not a multiple of Group interval it is coerced into one. For example, if your Group interval is 5 minutes, and your Repeat interval is 9 minutes, the Repeat interval is rounded up to the nearest multiple of 5 which is 10 minutes.
```

Comment:
```suggestion
Repeat interval must not only be greater than or equal to group interval, but also must be a multiple of Group interval. If Repeat interval is not a multiple of group interval it is coerced into one. For example, if your Group interval is 5 minutes, and your Repeat interval is 9 minutes, the Repeat interval is rounded up to the nearest multiple of 5 which is 10 minutes.
```

Reply:
so i'm taking out caps for all ui elements. i think either we bold them as ui elements or we take out the caps.

Reply:
not sure what you mean by "coerced" here?